acceptance_tests/features/support/env.rb

```diff
@@ -21,18 +21,18 @@
 
   Capybara.register_driver :selenium do |app|
     require 'selenium/webdriver'
-    profile = Selenium::WebDriver::Firefox::Profile.new
+    profile = Selenium::WebDriver::Chrome::Profile.new
     profile['browser.helperApps.alwaysAsk.force'] = false
     profile['browser.cache.disk.enable'] = false
     profile['browser.cache.memory.enable'] = false
-    Capybara::Selenium::Driver.new(app, :browser => :firefox, :profile => profile)
+    Capybara::Selenium::Driver.new(app, :browser => :chrome)
   end
 else
-  Capybara.run_server = false
-  Capybara.register_driver :poltergeist do |app|
-    Capybara::Poltergeist::Driver.new(app, :phantomjs => Phantomjs.path)
-  end
-  Capybara.default_driver = :poltergeist
+  Capybara.run_server = false
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, :phantomjs => Phantomjs.path)
+  end
+  Capybara.default_driver = :poltergeist
 end

```

Comment:
@joefitter I'm not sure why but I'm having the same issue with a PR for the phantomJS version gemfile change on RTM...